.github/workflows/build-docs.yml

```diff
@@ -6,9 +6,13 @@ name: Build docs
 on:
   push:
     branches: [main]
+    paths:
+    - 'docs/**'
```

Comment:
Not sure if this is exactly correct, since it looks like docstrings are used to generate the API docs.